spec/lib/slack_adapter_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'rails_helper'
+
+describe SlackAdapter do
+  subject { SlackAdapter.new }
+
+  describe "#channels_create" do
+    it "is a thin wrapper around Slack Gem's method" do
+      slack_double = double(channels_create: true)
+      stub_slack!(slack_double)
+      expect(slack_double).to receive(:channels_create)
+      subject.channels_create
+    end
+  end
+
+  describe "#send_team_invite" do
+    before do
+      allow(Time).to receive(:now).and_return(42)
+    end
+
+    it "executes a post with the right queries" do
+      email = "some_email"
+      payload = "_attempts=1&channels=C067NNHEY&email=#{email}&set_active=true&team_id=#{ENV['SLACK_TEAM_ID']}&token=#{ENV['SLACK_TEST_TOKEN']}"
+      proper_args = { method: :post, url: "https://datamarathontest.slack.com/api/users.admin.invite?t=42", payload: payload }
+      expect(RestClient::Request).to receive(:execute).with(proper_args).and_return({ok: true}.to_json)
```

Comment:
Line is too long. [103/80]<br>Space inside { missing.<br>Space inside } missing.